### PR TITLE
DF-2294 Set KOF Tracking cookie to 90 days

### PR DESCRIPTION
**Pull request by tonim42**
This cookie is used by the KOF (konjunkturforschungsstelle ETH) for an approved scientific study or the behavior of recruiters on Job-Room. The data generated in the trackingservice are strictly anonymized. Allowing the cookie to be stored for 90 days will enable the KOF to improve its analysis by being able to observe recurring users.

### CODE REVIEW

File: alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class KofTrackingService {
       this.trackingId = existingTrackingId;
     } else {
       const newTrackingId: string = uuid();
-      this.cookieService.set(TRACKING_COOKIE_KEY, newTrackingId);
+      this.cookieService.set(TRACKING_COOKIE_KEY, newTrackingId, 90);
```
Comment (seco-mgabor) [inline alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts:31]: "Magic number" must be avoided.
This hard-coded value must be replaced by a value that comes from a configuration file.

Comment (tonim42) [reply alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts:31]: You are right :)

But I won't have the time. I would be grateful if either you or @emina-str Emina could take care of it.

Comment (emina-str) [reply alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts:31]: @tonim42  I don't have time, working on highest prio ticket : https://alv-ch.atlassian.net/browse/DF-2283

Comment (tonim42) [reply alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts:31]: Ich verstehe. Wo wäre denn der richtige Ort für diese Konfiguration?

Comment (tonim42) [reply alv-portal-ui/src/app/shared/tracking/kof-tracking.service.ts]: @seco-mgabor  @emina-str  Ich erfasse ein Extraticket für die Konfiguration damit das sauber gemacht wird. Bitte darum diesen PR zu approven.

**Pull request merged**